mypy/suggestions.py

```diff
@@ -401,12 +401,6 @@ def get_default_arg_types(self, fdef: FuncDef) -> List[Optional[Type]]:
             for arg in fdef.arguments
         ]
 
-    def add_adjustments(self, typs: List[Type]) -> List[Type]:
-        if not self.try_text or self.manager.options.python_version[0] != 2:
```

Comment:
It looks like whole `--try-text` argument (Try using unicode wherever str is inferred, according to help) is not very useful now. So can probably be deleted (together with tests).